### ISSUE TITLE
[Bug]:The new mouse mapping fails frequently

### ISSUE THREAD

**Issue by ashran** [PlayCover/PlayCover#522]
### Describe the bug

When playing Genshin Impact, the new mouse mapping has fixed the lag issue, but it fails frequently. I need to press the Option key to bring up the mouse pointer and rotate the game screen to get it back to normal.

Version: Nightly build 149.

macOS: 13.

### Steps to reproduce

This bug happens quite often.

### Expected behaviour

Hope it gets fixed. The old mapping method is very laggy.

### What version of PlayCover are you using?

Nightly/beta

### What version of macOS are you using?

Ventura (macOS 13)

### Issue Language

- [X] Yes my issue is written in English

**Comment by wenboLee** [comment]
and keyboard also.

**Comment by zzz-one** [comment]
> and keyboard also.

with a lot of feedback

**Comment by IsaacMarovitz** [comment]
Presumed fixed in release 2.0.0+

**Comment by azaryabernard** [comment]
WIth mouse, I'm also noticing lag, stutter, and frequent fails BUT I've noticed that this issue only occurs when I'm using a higher polling rate mouse (for example using usb dongle). When I switch to using bluetooth mode (60Hz), the mouse keymapping works perfectly.

I've tried changing the sensitivity of both inside the PlayCover app or in-game and nothing seems to help.

[PlayCover 2.0.2]

**Comment by azaryabernard** [comment]
@IsaacMarovitz 
WIth mouse, I'm also noticing lag, stutter, and some fails BUT I've noticed that this issue only occurs when I'm using a higher polling rate mouse (for example when using usb dongle). When I switch to using bluetooth mode (60Hz), the mouse keymapping works perfectly, using the built-in trackpad also works perfectly.

I've tried changing the sensitivity of both inside the PlayCover app or in-game and nothing seems to help.

[PlayCover 2.0.2, MBP 16" M1 Pro base model, tested on Genshin Impact and Honkai Impact 3rd]
